### PR TITLE
4th and High Expanded Skill Rebalance (1st)

### PR DESCRIPTION
**Pull request by llchrisll**
Now after some consideration, I finally made a seperate branch for these skill rebalances.
They are also now merged with the normal skillinfoz files so they are ready to use.

The only thing left are the Expanded Classes.
But until rAthena doesn't officially have this skill rebalance merged, which will happens once this [PR](https://github.com/rathena/rathena/pull/7024) is done.

### CODE REVIEW

**Comment by reunite-ro** [comment]
If im not mistaken some of it already implemented/merged, I heard its being split up for easier track, maybe.
image

**Comment by llchrisll** [comment]
> If im not mistaken some of it already implemented/merged, I heard its being split up for easier track, maybe. image

Well I’m waiting for the PR directly to be closed/merged

File: Translation/Renewal/data/luafiles514/lua files/skillinfoz/skillinfolist.lub [hunk 1]
```diff
@@ -15521,11 +15521,11 @@ SKILL_INFO_LIST = {
 		bSeperateLv = false,
 		AttackRange = { 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 }
 	},
-	[SKID.SKE_RISING_SUN] = {
+	[SKID.[SKID.SKE_RISING_SUN] = {
```
Comment (Pokye) [inline Translation/Renewal/data/luafiles514/lua files/skillinfoz/skillinfolist.lub:15524]: Error
```[SKID.[SKID.SKE_RISING_SUN] = {```
Correct
```[SKID.SKE_RISING_SUN] = {```

**Comment by llchrisll** [comment]
Just noticed and fixed it on the 2nd rebalance files while testing. Thanks for pointing it out that it origins from this xD.

**Comment by llchrisll** [comment]
As I will manually merge the rebalances anyway, I will close them.

**Pull request closed without merging**